Newsletters: add 244 (2023-03-29)

harding opened the pull request:
- [x] Releases/RCs, topic links @harding 
- [x] Bitcoin SE @bitschmidty

vostrnad commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,347 @@
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #27278][] begins logging by default when a header for a
+  new block is received (unless the node is in Initial Block Download
+  (IBD)).  This was [inspired][obeirne selfish] by multiple node
```
Maybe avoid the ugly double parenthesis?
```suggestion
  new block is received unless the node is in Initial Block Download
  (IBD).  This was [inspired][obeirne selfish] by multiple node
```

vostrnad commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,347 @@
+    as any successful spending from an outdated state transaction will
+    prevent an outdated commitment transaction from being published
+    onchain.  Since the funds involved in that outdated state
+    transaction belong entirely to the party who is in breech of the
```
```suggestion
    transaction belong entirely to the party who is in breach of the
```

bitschmidty commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,377 @@
+    its initial state.  First, Alice and Bob can cooperatively close the
+    channel at any time by spending the funding transaction output (the same
+    as done in the current LN protocol).  Second, they could update the
+    state, for example increasing Alice's balance by 10M sat and decreasing
```
```suggestion
    state, for example, increasing Alice's balance by 10M sat and decreasing
```

bitschmidty commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,377 @@
+
+    With multiple states available, it's possible to accidentally or
+    deliberately close the channel in an outdated state.  For example, Bob may
+    try to close the channel in state S0 where he had an additional 10M
```
```suggestion
    try to close the channel in state _S<sub>0</sub>_ where he had an additional 10M
```

bitschmidty commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,377 @@
+    state is revoked by each party giving the other a witness[^keychain] for
+    spending the first output from their respective state transactions for
+    the previous state (S0).  Neither party can use the other's witness
+    because the S0 state transactions don't themselves contain witnesses yet
```
```suggestion
    because the _S<sub>0</sub>_ state transactions don't themselves contain witnesses yet
```

bitschmidty commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,377 @@
+    deliberately close the channel in an outdated state.  For example, Bob may
+    try to close the channel in state S0 where he had an additional 10M
+    satoshis.  To do this, Bob signs and broadcasts his state transaction
+    for S0.  Bob can't take any further action immediately because of
```
```suggestion
    for _S<sub>0</sub>_.  Bob can't take any further action immediately because of
```

bitschmidty commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,377 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.16.0-beta.rc3][] is a release candidate for a new major
```
[lnd v0.16.0-beta.rc5](https://github.com/lightningnetwork/lnd/releases/tag/v0.16.0-beta.rc5)

bitschmidty commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,377 @@
+  app in the BTCPay user interface.
+
+- [BIPs #1425][] assigns [BIP93][] to the codex32 scheme for encoding
+  [BIP32][] recovery seeds using Shamir's Secret Sharing (SSS)
```
Not sure that it matter but

```suggestion
  [BIP32][] recovery seeds using Shamir's Secret Sharing Scheme (SSSS)
```

harding replied:
Alliterative maximalism FTW.

harding commented on _posts/en/newsletters/2023-03-29-newsletter.md:
```diff
@@ -0,0 +1,377 @@
+    later time however he would like.
+
+    {:.center}
+    ![Alice spends honestly from the channel](/img/posts/2023-03-tunable-honest-spend.dot.png)
```
I'm pushing an edit for this as a separate commit in case you don't like it and want to revert before publication (otherwise it can be squashed like normal).

bitschmidty replied:
Dave, thanks I think the details adds clarity (even though also crowding)

harding commented:
Made edits for feedback (thanks @vostrnad and @bitschmidty !), reviewed @bitschmidty section (thanks!), updated releases/RCs and added topic links.  Thanks for reading through my long section about tunable penalties!

bitschmidty commented:
Ensured review comments were addressed and resolved. Reviewed topic links and fixups from @harding. Squashed. Thanks @harding and @vostrnad for your contributions this week! 🚀
